src/content_script.js

```diff
@@ -218,7 +224,30 @@ function addFacebookBadge (target, badgeClassUId, socialAction) {
       document.querySelector(".fbc-has-badge.js-fbc-prompt-active").classList.remove("js-fbc-prompt-active");
       e.target.parentElement.parentNode.parentNode.classList.remove("active");
     });
-  } else if (socialAction === "share") {
+  } else if (socialAction === "share-passive") {
```

Comment:
This `else-if` is duplicated below for the share statement. Recommend creating function both can share.

Reply:
Resolved in 6d0df08431eac294f1cd867114926fc77f994e2e